benchmarks/dynamo/common.py

```diff
@@ -1091,6 +1092,10 @@ def deepcopy_and_maybe_ddp(model):
             model = copy.deepcopy(model)
             if self.args.ddp:
                 model = DDP(model, find_unused_parameters=True)
+            elif self.args.fsdp:
+                model = FSDP(model)
```

Comment:
Just want to understand here -- this is a single wrap, correct?

Reply:
Did we decide that we're not doing any custom wrapping?

Reply:
no, we still are. this test is used for sweeping all of the benchmark suite models.  for that purpose it’s not feasible to configure custom wrapping, but the sweep helps find bugs with dynamo handling of those models inside FSDP wrapper. The main case i expect to catch is that UnspecializedNNModule misses some feature used by a model.